File: Back/src/FFStats.Domain/Models/Jogador.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace FFStats.Domain.Models
{
    [Table("tb_jogador")]
    public class Jogador
    {
       public int id { get; set; }
       public string jogadorNome { get; set; }
       public string jogadorNick { get; set; }
       public int idJogo { get; set; }
       // Foreign key for Line
       public int lineId { get; set; }
       // Navigation property for Line
       [ForeignKey("lineId")] 
       public Line Line{ get; set; }
       // Navigation property for PartidasJogadores
       public IEnumerable<PartidaJogador> PartidasJogadores { get; set; }

    }
}

File: Front/FFStats-App/src/app/Model/Jogador.ts
import { Line } from "./Line";
import { Partida } from "./Partida";

export interface Jogador {
  id: number;
  jogadorNome: string;
  jogadorNick: string;
  idJogo: number;
  lineId: number;
  line: Line;
  partidasJogadores: Partida[];
}
